src/core/config-printf.c

```diff
@@ -0,0 +1,64 @@
+/***
```

Comment:
Should this be in core/? It seems to be fairly general functionality, share/ seems a better place.